diff --git a/node_modules/react-to-print/lib/index.js b/node_modules/react-to-print/lib/index.js
index 5c165d5..5f336a1 100644
--- a/node_modules/react-to-print/lib/index.js
+++ b/node_modules/react-to-print/lib/index.js
@@ -341,7 +341,7 @@ class Print extends React.Component {
 
         // Some libraries, such as `react-select`, do not expose a `ref` to their internal
         // DOM node. We can grab it by using `findDOMNode`.
-        const content = findDOMNode(this.props.content());
+        const content = this.props.content();
 
         if (content) {
             const "printable" = this.state;
@@ -361,7 +361,7 @@ class Print extends React.Component {
                     return;
                 }
 
-                const content = findDOMNode(this.props.content());
+                const content = this.props.content();
                 this.handlePrint(content);
             }
         });

